pkg/scheduler/apis/config/v1beta2/default_plugins.go

```diff
@@ -76,6 +76,8 @@ func getDefaultPlugins() *v1beta2.Plugins {
 				{Name: names.PodTopologySpread},
 				{Name: names.TaintToleration},
 				{Name: names.NodeAffinity},
+				{Name: names.NodeResourcesFit},
+				{Name: names.NodeResourcesBalancedAllocation},
```

Comment:
More importantly, we need to do this for `apis/config/v1`

Reply:
I think this already in Multi

Reply:
ah, true